controllers/ccruntime_controller.go

```diff
@@ -956,6 +942,14 @@ func (r *CcRuntimeReconciler) removeNodeLabels(nodesList *corev1.NodeList) (ctrl
 		return ctrl.Result{}, err
 	}
 
+	kataCleanupDoneLabel := make([]string, 0, len(r.ccRuntime.Spec.Config.UninstallDoneLabel))
+	for key := range r.ccRuntime.Spec.Config.UninstallDoneLabel {
+		kataCleanupDoneLabel = append(kataCleanupDoneLabel, key)
+	}
+	if len(kataCleanupDoneLabel) != 1 {
+		return ctrl.Result{}, fmt.Errorf("UninstallDoneLabel must only have one entry")
+	}
+
```

Comment:
this could also just take the first element and print a warning about ignored elements if `len(r.ccRuntime.Spec.Config.UninstallDoneLabel) > 1`?

Reply:
I think so as well but this is how it's treated everywhere else...

Reply:
OK, I'm fine as it is then. Resolving this